logstash-output-lumberjack.gemspec

```diff
@@ -24,7 +24,12 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
 
   s.add_runtime_dependency 'jls-lumberjack', ['>=0.0.23']
+  s.add_runtime_dependency "stud"
```

Comment:
I wish `bundle install` would also yield theses errors since he is using the gemspec to install the depedencies, TIL to always test with `gem build` my changes to a gemspec.
![bart-simpson-generator](https://cloud.githubusercontent.com/assets/640/9331034/04fe93e8-458b-11e5-824a-86d5604ee74b.gif)